README.md

```diff
@@ -312,6 +312,55 @@ $response->hasMore; // false
 $response->toArray(); // ['object' => 'list', 'data' => [...], ...]
 ```
 
+#### `function tool`
```

Comment:
Can we remove the header (since it makes it seem like its a top-level command) and do a simple line beneath the create call?

<img width="1046" height="716" alt="Image" src="https://github.com/user-attachments/assets/66cbf403-97c6-42c3-aab3-8b11d8543131" />

Much like we did in Chat